README.md

```diff
@@ -133,6 +133,19 @@ submitToTreeherder(
 )
 ```
 
+## testProject
+Queries the Service Book project API for the given project `name`, iterates over
+its associated test repositories, checks them out from SCM, and executes their
+```run``` file(s).  Finally, it returns ```exit 0``` on successful/passing
+tests, and ```exit 1``` in the event of failed builds.
+
+## Examples
+```@Library('fxtest@1.7') _
+
+def sb = new org.mozilla.fxtest.ServiceBook()  
+sb.testProject('kinto')
+
```

Comment:
You need to close the code sample with <code>```</code>, otherwise the next heading is considered code.

Reply:
Fixed; thanks (and, oops!)